test/JDBC/input/BABEL-CASE_EXPR-vu-prepare.sql

```diff
@@ -45,4 +45,112 @@ BEGIN
         END
     );
 END;
-GO
\ No newline at end of file
+GO
+
+CREATE TABLE BABEL_5341_T1 (
+    COL1 NUMERIC(20,2),
+    COL2 DECIMAL(20,2),
+    COL3 NUMERIC(38,0),
+    COL4 DECIMAL(38,0),
+    COL5 NUMERIC(2,0),
+    COL6 DECIMAL(2,0)
+);
+GO
+
+INSERT INTO BABEL_5341_T1 (COL1, COL2, COL3, COL4, COL5, COL6)
+VALUES (12345.67, 54321.98, 1234567890123456789, 9876543210987654321, 42, 99);
+GO
+
+INSERT INTO BABEL_5341_T1 (COL1, COL2, COL3, COL4, COL5, COL6)
+VALUES (98765.43, 12345.67, 9876543210987654321, 1234567890123456789, 10, 50);
+GO
+
+INSERT INTO BABEL_5341_T1 (COL1, COL2, COL3, COL4, COL5, COL6)
+VALUES (11111.11, 22222.22, 1111111111111111111, 2222222222222222222, 11, 22);
+GO
+
+INSERT INTO BABEL_5341_T1 (COL1, COL2, COL3, COL4, COL5, COL6)
+VALUES (33333.33, 44444.44, 3333333333333333333, 4444444444444444444, 33, 44);
+GO
+
+INSERT INTO BABEL_5341_T1 (COL1, COL2, COL3, COL4, COL5, COL6)
+VALUES (55555.55, 66666.66, 5555555555555555555, 6666666666666666666, 55, 66);
+GO
+
+CREATE VIEW BABEL_5341_V1 AS 
```

Comment:
in verify part, we should check the precision, scale of the column using some system metadata views